src/reducers/patients.js

```diff
@@ -0,0 +1,13 @@
+import { FETCH_PATIENTS_FULFILLED } from '../actions/types';
+
+export default function(state = {patientEntries: [], meta:{}}, action)  {
+  switch (action.type) {
+    case FETCH_PATIENTS_FULFILLED:
+      const patientEntries = action.payload.data.entry.map((e) => e.resource);
+      const { total, link } = action.payload.data;
+      const meta = { total,link };
```

Comment:
Ah I see. We should try to find a way to make that more clear. Perhaps the pieces of state should be `patient.pagedPatients` and `patient.totalPatientCount`? Is `link` used or do we have plans to use it?